skillmap/src/components/SkillCard.tsx

```diff
@@ -62,6 +70,9 @@ export class SkillCardImpl extends React.Component<SkillCardProps> {
                 <div className="skill-card-label">
                     <div className="skill-card-title">
                         {(status === "locked" || status === "completed") && <i className={`icon ${status === "locked" ? "lock" : "check circle"}`} />}
+                        {(status === "inprogress" || status === "notstarted") && maxSteps &&
```

Comment:
so "notstarted" shows 0/maxSteps?

Reply:
oh no it doesn't but this is definitely confusing so I'll change it!

Reply:
ohh notstarted doesn't have maxsteps, i see